index.html

```diff
@@ -3,34 +3,54 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="assets/main.css">
+  <link rel="stylesheet" 
+        href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" 
+        integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" 
+        crossorigin="anonymous">
   <title>Document</title>
 </head>
 <body>
-  <h2 class = 'bk-store-header'>Book Store</h2>
-
   <div class="container">
-    <div class="card">
-      Author, title, pages
-    </div>
-  </div>
+    <h2 class = 'display-4 text-center'>Book Store</h2>
 
-  <div class="container library grid-container"> 
-    <div class="form">
-      <form action="javascript:;" onsubmit="myFunction(this)"> 
-        <label for="author">Author:</label><br>
-        <input type="text" name="author" value="">
-        <br>
-        <label for="title">Title:</label><br>
-        <input type="text" name="title" value="">
-        <br>
-        <label for="number">Pages:</label><br>
-        <input type="number" name="pages" value="">
-        <br>
-        <input type="button" name="button" value="Click" onClick="addBookToStore(this.form)">
-      </form>
-    </div>
+    <div class="d-flex">
+        
+            <form id = "book-form" class="mr-5" autocomplete="off">
+                <div class="form-group">
+                  <label for="author">Author:</label>
+                  <input type="text" class="form-control" name="author" id="author">
+                </div>
+                  
+                <div class="form-group">
+                  <label for="title">Title:</label>
+                  <input type="text" class="form-control" name="title" id="title">
+                </div>
+
+                <div class="form-group">
+                  <label for="pages">Pages:</label>
+                  <input type="text" class="form-control" name="pages" id="pages">
+                </div>
+              
+                <input type="submit" class="btn btn-primary btn-block" value="Click">
+            </form>
```

Comment:
This form should also contain a checkbox to set if I read the book before I add it to the library
![image](https://user-images.githubusercontent.com/73254040/98422153-925d6980-2050-11eb-8099-6ca50aef418f.png)